policyengine_us/parameters/gov/contrib/congress/tlaib/income_security_package/end_child_poverty_act/filer_credit/phase_out/rate.yaml

```
@@ -0,0 +1,8 @@
+description: Representative Rashida Tlaib's End Child Poverty Act applies this filer credit phase-out rate.
```

```suggestion
description: Representative Rashida Tlaib's End Child Poverty Act is phased out at the following rate of [].
```

Fill in income definition etc.